readers/cassandra/messages_test.go

```diff
@@ -36,6 +36,8 @@ var (
 	vb          = true
 	vd          = "base64"
 	sum float64 = 42
+
```

Comment:
This `var` block looks like it's actually `const`. The only var is `uuidProvider`.

Reply:
Not really, we need `var` to use the address (error is `cannot take the address of v`). `addr` is also reassigned in `setup_test`